intercom/templatetags/intercom.py

```diff
@@ -101,7 +101,7 @@ def intercom_tag(context):
                     log.warning("%s couldn't be imported, there was an error during import. "
                                 "skipping. %s", custom_data_class, e)
 
-            custom_data = json.dumps(custom_data)
+            custom_data = custom_data
```

Comment:
@kencochrane Great, thanks. Will do the changes and change the Javascript code too, I just had another interaction with Intercom support about updating the client side code.